guidelines/mwm-sadd.md

```diff
@@ -21,40 +21,54 @@ The scope of this document is limited to the design of the MONAI Workload Manage
 | Term        | Definition                                                                                                                                                       |
 | ----------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | Export Sink | An export sink is a user-configured sink where the results (application-generated artifacts) are assigned to and later picked up by the export service clients. |
-| MWM         | MONAI Workload Manager                                                                                                                                           |
+| MWM         | MONAI Deploy Workload Manager                                                                                                                                           |
 | MIG         | MONAI Informatics Gateway                                                                                                                                        |
 
 ## Reference Documents
 
-- [MONAI Workload Manager Requirements](mwm-srs.md)
+- [MONAI Deploy Workload Manager Requirements](mwm-srs.md)
 
 ## Architecture Details
 
-MONAI Workload Manager includes a set of services and APIs to interconnect other subsystems in MONAI Deploy. Every service in the MWM runs on one or more threads that may receive requests from external systems/services. It also interacts with a database and other external services, such as the App Server or Argo, for job management.
+MONAI Deploy Workload Manager is divided into three separate services where each includes a set of services and APIs to interconnect other subsystems in MONAI Deploy. Every service in the MWM runs on one or more threads that may receive requests from external systems/services. It also interacts with a database and other external services, such as the App Server or Argo, for job management.
```

Comment:
Three separate services and then each includes a set of services? Are they really `services` in the common understanding or simply component? Reads like overload of the term `service`.